content/dst/waku/waku-scaling.md

```diff
@@ -0,0 +1,163 @@
+
+We will support the Conduit of Expertise narrative directly by providing valuable insights to Waku and the ability to theorise, reason about, test, measure and improve the performance, stability and scalability of Waku. 
+
+These efforts will contribute in these ways to the Conduit of Expertise narrative:
+
+* Accelerate improvements to Waku, improving the developer community's experience and satisfaction both inside and outside of IFT's ecosystem, through allowing repeatable, measureable and real world insights into Waku, all the way from theory to practice and back.
+* Improve the RFC culture by allowing for faster and easier development of RFCs with the aid of rapidly accelerated insights into how an RFC in development will perform as it's being expanded and going through the draft process.
+* Allow easier post-mortem analysis of the success or relative performance of a given RFC - does this change use more or less bandwidth? Did it improve things? Seeing the effects of changes at scale allows for a greater ability to usefully wrap up work on and conclude an RFC process and document and absorb what we learned in the process into further improvements.
+
+## Task List
+
+### High Scalability Waku Demonstration
+
+* fully qualified name: `vac:dst:waku:waku-scaling`
+* owner: Wings
+* status: 95%
+* start-date: 2024/03/01
+* end-date: 2024/11/01
+
+#### Description
+Demonstrate a working, real world, large scale Waku network.
+
+Measure its performance and attempt to support the assertion that Waku is a scalable solution that can work in networks at sizes that push the limits of what the theoretical work we did predicted is possible.
+
+Specifically, we want to deploy a 10,000 node Waku network and measure its performance in terms of message delivery, bandwidth usage, and other metrics. We want to deliver a report on what we learned, what we tested and what we found.
+
+The report should include analysis of the performance of Waku at extreme scale, providing insights that allow people to see significant supporting evidence that Waku can in fact scale to these sizes and perform reliably.
+
+#### Deliverables
+
+- [x] https://github.com/vacp2p/10ksim - A working set of bundled and compatible Kubernetes manifests that allow for up to a 10,000 node Waku network to be reliably created and measured. The manifests should be compatible with [[vac|dst|deployer-tool|deployer-tool]] and flexible.
+
+- [ ] A useful set of measurements taken with the monitoring system and tooling we have available.
+
+- [ ] The monitoring system stays stable the entire time, providing useful information and metrics.
+
+### Test Store Protocol At Scale
+
+* fully qualified name: `vac:dst:waku:waku-scaling:test-store-protocol-at-scale`
+* owner: Alberto
+* status: 0%
+* start-date: 2024/09/02
+* end-date: 2024/12/31
```

Comment:
If I am not wrong, all milestones that fall under the Waku scaling milestone are due to 2024/11/1. 
I mean Bandwidth, Reliability, Discv5, Store, Shards, Gowaku+Nwaku, Filter+Lightpush, peer exchange protocol and resource-restricted devices.

Reply:
We should push to make a schedule/timeline that fits properly.